ArchiveDropView.m

```diff
@@ -61,21 +61,25 @@ - (NSImage *)image
 
 - (void)logError:(NSString*) message
 {
+    printf("\x1b[1;91m%s\x1b[0m\n", message.UTF8String);
```

Comment:
I think we should remove these `printfs`. I believe these are here to show output when running from the command line version - they might be better to just be in the logMessage function and not do the colours as we're assuming the colour scheme of the terminal.